src/routes/diluent.test.js

```diff
@@ -0,0 +1,89 @@
+import express from 'express';
+import request from 'supertest';
+import passport from 'passport';
+import AnonymousStrategy from 'passport-anonymous';
+import bodyParser from 'body-parser';
+
+import diluent from './diluent';
+import database from '../modules/database';
+
+describe('diluent route resource', () => {
+  const app = express();
+
+  passport.use(new AnonymousStrategy());
+  app.use(bodyParser.json());
+  app.use(bodyParser.urlencoded({ extended: false }));
+  app.use(diluent);
+
+  afterAll(() => {
+    database.sequelize.close();
+  });
+
+  const date = new Date();
```

Comment:
Don't need this if you're using sequelize-mock

Reply:
done